azure/table_azure_security_center.go

```diff
@@ -0,0 +1,168 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/resources/mgmt/2020-03-01-preview/policy"
+	"github.com/Azure/azure-sdk-for-go/services/preview/security/mgmt/v1.0/security"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+)
+
+//// TABLE DEFINITION ////
+
+func tableAzureSecurityCenter(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azure_security_center",
+		Description: "Azure security Center",
+		List: &plugin.ListConfig{
+			Hydrate: listSecurityCenter,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "setting",
+				Description: "",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "pricing",
+				Description: "",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "auto_provisioning",
+				Description: "",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "contact",
+				Description: "",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "policy",
+				Description: "",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Standard columns
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromConstant("Security Center"),
+			},
+			{
+				Name:        "akas",
+				Description: ColumnDescriptionAkas,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Id").Transform(idToAkas),
+			},
+			{
+				Name:        "subscription_id",
+				Description: ColumnDescriptionSubscription,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Id").Transform(idToSubscriptionID),
+			},
+		},
+	}
+}
+
+//// FETCH FUNCTIONS ////
+
+func listSecurityCenter(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	session, err := GetNewSession(ctx, d, "MANAGEMENT")
+	if err != nil {
+		return nil, err
+	}
+	subscriptionID := session.SubscriptionID
+
+	settingClient := security.NewSettingsClient(subscriptionID, "")
+	settingClient.Authorizer = session.Authorizer
+
+	pricingClient := security.NewPricingsClient(subscriptionID, "")
+	pricingClient.Authorizer = session.Authorizer
+
+	autoProvisionClient := security.NewAutoProvisioningSettingsClient(subscriptionID, "")
+	autoProvisionClient.Authorizer = session.Authorizer
+
+	contactClient := security.NewContactsClient(subscriptionID, "")
+	contactClient.Authorizer = session.Authorizer
+
+	PolicyClient := policy.NewAssignmentsClient()
+	PolicyClient.Authorizer = session.Authorizer
```

Comment:
Can we add separate functions for each call? and finally combine the outputs and return from `listSecurityCenter`?